### ISSUE TITLE
Cellpose - Stacks (2D+t) segmentation bug

### ISSUE THREAD

**Issue by xcarmengrandix** [HenriquesLab/ZeroCostDL4Mic#149]
Hi, 
I'm testing segmentation with Cellpose 2D (version of the notebook 1.14). I trained a model from scratch and it's working great, but when I try to segment stack (2D+t) images, the last time points are not segmented correctly, since many cells are given the same label (the white one) and recognized then as a single object:
![image](https://user-images.githubusercontent.com/87519043/139086290-e35d4099-03c2-40a8-8136-e58d7a8138ea.png)

I thought the reason of this was the presence of too many cells in my image but if I segment this specific time point as a single image then the segmentation works nicely: 
![image](https://user-images.githubusercontent.com/87519043/139086511-32803f94-5139-4709-8d94-824e5fbeb55a.png)

Hope this will help to improve the notebook.

Carmen

**Comment by guijacquemet** [comment]
Dear @xcarmengrandix 

Many thanks for reaching out!
I cannot reproduce the bug on my side with my test data. Would it be possible for you to share with me the model you are using and the stack that is causing issues?

If not, could you tell me a little bit more about the issue? Do you still get the same error when using one of the pre-trained models (ie Cyto2)?

What is the pixel intensity value of the large object that contains multiple cells?

Thanks

Guillaume

**Comment by xcarmengrandix** [comment]
Hi @guijacquemet,
thanks for the quick answer! I couldn't reproduce the bug anymore either this morning and I found out that it was due to the conversion of the 16-bit image into 8-bit in ImageJ. So, when analysing stack images I still get an output as shown above, with many white labels, but these are still recognized as different ROIs if the image is kept in a 16-bit format. 

Sorry, my bad!

Can I take advantage to ask here if you are planning to implement soon the section for the loss of function inspection? Your script works great and I'm planning to use it for the analysis of my experiments :)

Thanks!
Carmen

**Comment by guijacquemet** [comment]
Hi @xcarmengrandix 
Great! Glad you worked out what the problem was!

For the loss of function inspection, difficult to give you a timing, It's on the list of things we want to implement but we currently do not have anyone to work on this.
